Cargo.toml

```diff
@@ -9,7 +9,7 @@ build = "build.rs"
 
 [dependencies]
 tokio = { version = "1", features = ["full"] }
-warp = "0.3"
+axum = { version = "0.6.0-rc.2", features = ["json"] }
```

Comment:
Why pre-release version?

Reply:
There were docs for it and it's in `rc` so wanted to save future refactors once it becomes stable.